ml-conversational-analytic-tool/run.py

```diff
@@ -2,14 +2,30 @@
 # SPDX-License-Identifier: Apache-2.0
 
 import argparse
+import tarfile
+import os
+
 from sklearn.model_selection import train_test_split
 
 from baseCNN import BaseCNN
 from baseLSTM import BaseLSTM
 from preProcessedDataset import PreProcessedDataset
 
-
-def run(annotated_filename, dataset_filename, outcome, encoding_type, model_type, padding):
+model_direcrtory = 'models'
+model_version = '0001'
```

Comment:
can we use semantic versioning?

Reply:
semantic versioning sounds good, I agree we should use it. I took that `0001` from some tensorflow kserve examples. I'll  test kserve with `0.0.1` and if it works, will change that to semantic versioning.

Reply:
Actually, thinking about this more.. would it be better for the user to pass the version in until (if/when) we decide to support the automatic versioning?

Reply:
I've tested with semantic model version, but looks like the tenserflow predictor does not support it. The version string should be an integer. I looked at the predictor code, that limitation is in [tensorflow_serving](https://github.com/tensorflow/serving/blob/master/tensorflow_serving/sources/storage_path/file_system_storage_path_source.cc ) function
```
// Converts the string version path to an integer.
// Returns false if the input is invalid.
bool ParseVersionNumber(const string& version_path, int64_t* version_number) {
  return strings::safe_strto64(version_path.c_str(), version_number);
}
```
We should stick with `0001` version format for now, and may be submit an issue and PR to tenserflow for model semantic versioning support ?

Reply:
sounds good - but can we pass the version in instead of providing it?

Reply:
oops sorry, didn't look at that new changes. All good! thank you!